src/Command/Ingester/CreateUserCommand.php

```diff
@@ -0,0 +1,461 @@
+            's',
+            InputOption::VALUE_REQUIRED,
+            'Comma separated list of line item slugs',
+        );
+
+        $this->addOption(
+            'batch',
+            'b',
+            InputOption::VALUE_REQUIRED,
+            'User Create Batch size',
+            self::DEFAULT_BATCH_SIZE
+        );
+
+        $this->addArgument(
+            'user-prefix',
+            InputArgument::REQUIRED,
+            'user prefix list'
+        );
+
+        $this->addOption(
+            self::OPTION_GROUP_PREFIX,
+            'p',
+            InputOption::VALUE_REQUIRED,
+            'Group Prefix',
+        );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->symfonyStyle = new SymfonyStyle($input, $output);
+        $this->symfonyStyle->title('Simple Roster - Automate the user-generation');
+
+        if (!empty($input->getOption(self::OPTION_LINE_ITEM_IDS)) && !empty($input->getOption(self::OPTION_LINE_ITEM_SLUGS))) {
+            throw new InvalidArgumentException(
+                sprintf(
+                    "Option '%s' and '%s' are exclusive options.",
+                    self::OPTION_LINE_ITEM_IDS,
+                    self::OPTION_LINE_ITEM_SLUGS
+                )
+            );
+        }
+
+        if ($input->getOption(self::OPTION_LINE_ITEM_IDS)) {
+            $this->initializeLineItemIdsOption($input);
+            $criteria = $this->getFindLineItemCriteria();
+            $lineItemsCollection = $this->lineItemRepository->findLineItemsByCriteria($criteria);
+            if(!empty($lineItemsCollection)) {
+                $slugArr = [];
+                foreach($lineItemsCollection as $lineVal){
+                    array_push($slugArr, $lineVal->getSlug());
+                }
+                if(!empty($slugArr)) {
+                    $this->lineItemSlugs = $slugArr;
+                }
+            }
+        }
+
+        if ($input->getOption(self::OPTION_LINE_ITEM_SLUGS) && empty($this->lineItemSlugs)) {
+            $this->initializeLineItemSlugsOption($input->getOption(self::OPTION_LINE_ITEM_SLUGS));
+        } 
+        
+        if (empty($this->lineItemSlugs)) {
+            $this->getAllLineItemSlugs();
+        }
+
+        if ($input->getArguments('user-prefix')) {
+            $this->initializeUserPrefixOption($input);
+        }
+        
+        $this->batchSize = (int)$input->getOption('batch') ?? self::DEFAULT_BATCH_SIZE;
+        
+    }
+
+    /**
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->symfonyStyle->comment('Executing Automation...');
+
+        try {
+            
+            $lineItems = $this->lineItemRepository->findAllAsCollection();
+            if ($lineItems->isEmpty()) {
+                throw new LineItemNotFoundException("No line items were found in database.");
```

Comment:
please avoid using double quotes if it is not necessary here and below

Reply:
Done